spyder/widgets/pathmanager.py

```diff
@@ -202,20 +215,52 @@ def setup(self):
         self.refresh()
 
     @Slot()
-    def synchronize(self):
+    def import_pythonpath(self):
+        """Import from PYTHONPATH environment variable"""
+        env_pypath = os.environ.get('PYTHONPATH', '')
+        if not env_pypath:
+            return
+
+        env_pypath = env_pypath.split(os.pathsep)
+        env_pypath_msg = '<br>'.join(env_pypath)
+        answer = QMessageBox.question(
+            self,
+            _("Import"),
+            _("Do you want to import the following paths into "
+              "PYTHONPATH Manager?"

```

Comment:
```suggestion
            _("Do you want to import the contents of your <tt>PYTHONPATH</tt> "
              "environment variable into Spyder? The following paths will be "
              "imported:"
```

* I think we can be clearer and more explicit here.
* I like that users can check the paths that will be imported. Thanks for adding that.